charts/tfy-agent/values.yaml

```diff
@@ -435,11 +447,38 @@ tfyAgentProxy:
   clusterRole:
     ## @param tfyAgentProxy.clusterRole.enable Create cluster role.
     enable: true
-    ## @param tfyAgentProxy.clusterRole.rules [array] Cluster role Authz rules.
-    rules:
+    ## @param tfyAgentProxy.clusterRole.clusterScopedAccessRules [array] Cluster scoped access rules for the control plane.
+    ## The control plane will always have cluster-wide access to the resources below.
+    clusterScopedAccessRules:
+      - apiGroups: ["storage.k8s.io"]
+        resources: ["storageclasses"]
+        verbs: ["get", "list"]
+
+      - apiGroups: ["networking.istio.io"]
+        resources: ["virtualservices"]
+        verbs: ["list"]
```

Comment:
Is this because we want to check against all routes for conflicts in the cluster instead of just the ones in allowed namespaces?

Reply:
Yes. We will later do the same check by using the DB.